src/rust-crypto/verification.ts

```diff
@@ -40,13 +41,16 @@ export class RustVerificationRequest
     extends TypedEventEmitter<VerificationRequestEvent, VerificationRequestEventHandlerMap>
     implements VerificationRequest
 {
+    /** a reëmitter which relays VerificationRequestEvent.Changed events emitted by the verifier */
```

Comment:
`ë`?

Reply:
https://www.merriam-webster.com/words-at-play/mary-norris-diaeresis

my use of it is tongue-in-cheek. Will replace if you find it distracting/confusing.